libs/core/algorithms/include/hpx/parallel/algorithms/detail/reduce_deterministic.hpp

```diff
@@ -65,6 +66,69 @@ namespace hpx::parallel::detail {
         }
     };
 
+    template <typename ExPolicy>
+    struct sequential_reduce_deterministic_rfa_t final
+      : hpx::functional::detail::tag_fallback<
+            sequential_reduce_deterministic_rfa_t<ExPolicy>>
+    {
+    private:
+        template <typename InIterB, typename T>
+        friend constexpr hpx::parallel::detail::rfa::
+            ReproducibleFloatingAccumulator<T>
+            tag_fallback_invoke(sequential_reduce_deterministic_rfa_t,
+                ExPolicy&&, InIterB first, std::size_t partition_size, T init,
+                std::true_type&&)
+        {
+            // __rfa_bin_host_buffer__ should be initialized by the frontend of
+            // this method
+
+            hpx::parallel::detail::rfa::ReproducibleFloatingAccumulator<T> rfa;
```

Comment:
Nitpick: could you please use `snake_case` identifier names instead of the `CamelCase` ones?